src/Twig/Extension/IconExtension.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class IconExtension extends AbstractExtension
+{
+
+    public function getFilters(): array
+    {
+        return array(
+            new TwigFilter('parse_icon', array($this, 'parseIcon')),
+        );
+    }
+
+    public function parseIcon(string $icon, bool $inline = false): string
+    {
+        switch (substr($icon,0,3)) {
+            case "<i ":
+                // full style is used by dev: '<i class="fa fa-plus"></i>'
+                if ($inline) {
+                    preg_match('/"([^"]+)"/', $icon, $output);
```

Comment:
Not sure it works everytime.

I could use '<i aria-hidden="true" class="fa fa-plus"></i>', you'll get `true` no ?
And what if I use `"<i class='fa fa-plus'></i>"` ?